Contest_API.md

```diff
@@ -1971,14 +1971,14 @@ The following endpoints are associated with commentary:
 
 Properties of award objects:
 
-| Name          | Type          | Description
-| :------------ | :------------ | :----------
-| id            | ID            | Identifier of the commentary.
-| time          | TIME          | Time of the commentary message.
-| contest\_time | RELTIME       | Contest time of the commentary message.
-| message       | string        | Commentary message text. May contain special tags for [teams](#teams) and [problems](#problems) on the format `#t<team ID>` and `#p<problem ID>` respectively.
-| team\_ids     | array of ID ? | JSON array of [team](#teams) IDs the message is related to.
-| problem\_ids  | array of ID ? | JSON array of [problem](#problems) IDs the message is related to.
+| Name          | Type        | Description
+| :------------ | :---------- | :----------
+| id            | ID          | Identifier of the commentary.
+| time          | TIME        | Time of the commentary message.
+| contest\_time | RELTIME     | Contest time of the commentary message.
+| message       | string      | Commentary message text. May contain special tags for [teams](#teams) and [problems](#problems) on the format `#t<team ID>` and `#p<problem ID>` respectively.
+| team\_ids     | array of ID | JSON array of [team](#teams) IDs the message is related to.
+| problem\_ids  | array of ID | JSON array of [problem](#problems) IDs the message is related to.
```

Comment:
If these team/problem IDs properties are listed in `/access`, then I think they should always contain an empty array instead of `null`.

Reply:
I'm torn about this one.

Reply:
I think this is the same as for the entry point thing: it doesn't really save a lot IMHO to make this nullable.